modules/packages/LinearAlgebra.chpl

```diff
@@ -823,6 +825,11 @@ proc matPow(A: [], b) where isNumeric(b) {
   if !isSquare(A) then
     halt("Array not square");
 
+  if (b < 0 && usingLapack) {
```

Comment:
The compiler will not evaluate this to false at compile time today, causing issues when LAPACK is not there, so the param flag should come first in this statement `usingLapack && b<0`

Reply:
My idea was to let users decide if they don't want negative matPow support then it's fine to not include lapack. So that's why I updated the comment as it will generate a "runtime error" and I thought it would be more flexible that way.

Reply:
Running the most basic local start_test
- `usingLAPACK`, check capitalization
- The issue is not about that, you don't want this block to be compiled if `usingLAPACK==false`. And currently, `b<0 && usingLAPACK` is evaluated at runtime, whereas `usingLAPACK && b<0` is evaluated at compile time, so compiler would know not to compile this block.